File: examples/traffic_encryption/connect.lua
local net_box = require('net.box')

local conn = net_box.connect({
    uri = 'admin:secret-cluster-cookie@localhost:3301',
    params = {
        transport = 'ssl',
        ssl_cert_file='./bootstrap/client-cert.pem',
        ssl_key_file='./bootstrap/client-key.pem',
        ssl_ca_file='./bootstrap/ca-cert.pem'
    }
})

local res = conn:eval("return 1+1")

print(res)

File: examples/traffic_encryption/python/connect.py
import tarantool

con = tarantool.Connection(
    'tarantool-router-msk',
    3301,
    user="admin",
    password="secret-cluster-cookie",
    transport='ssl',
    ssl_key_file='certs/tarantool/client-key.pem',
    ssl_cert_file='certs/tarantool/client.pem',
    ssl_ca_file='certs/ca/root-ca.pem',
    ssl_password='54321',
    connection_timeout=0.5,
    socket_timeout=0.5,
)

print(con.eval("return box.info.version"))

con.close()
